ni_measurementlink_generator/ni_measurementlink_generator/template.py

```diff
@@ -104,6 +111,23 @@ def _resolve_service_class(service_class: str, display_name: str) -> str:
     "--directory-out",
     help="Output directory for measurement files. Default: '<current_directory>/<display_name>'",
 )
+@click.option(
+    "-c",
+    "--collection",
+    default="",
+    help='\b\nThe collection that this measurement belongs to.'
+    + '\n-Expected format: "." delimited namespace/hierarchy case-insensitive string'
+    + '\n-Example: "CurrentTests.Inrush"',
```

Comment:
What is `\b\n` supposed to do? It doesn't seem to do anything.

Is the dash in `\n-Expected` intended to be a bullet? It doesn't look like one. 

The help for the other options uses `Default: x` without a line break or bullet, so I think you should remove the newlines and dashes. 

The help for the other options displays single quotes, not double quotes.

Example to add 2 tags has an odd number of double quotes.

```
Options:
  --measurement-version TEXT  Version number in the form 'x.y.z.q'. Default:
                              '1.0.0.0'
  -u, --ui-file TEXT          Name of the UI File. Default:
                              '<display_name>.measui'
  -s, --service-class TEXT    Service Class that the measurement belongs to.
                              Default: '<display_name>_Python'
  -D, --description TEXT      Short description of the measurement
  -d, --description-url TEXT  Description URL that links to a web page
                              containing information about the measurement.
  -o, --directory-out TEXT    Output directory for measurement files. Default:
                              '<current_directory>/<display_name>'
  -c, --collection TEXT       The collection that this measurement belongs to.
                              -Expected format: "." delimited namespace/hierarchy case-insensitive string
                              -Example: "CurrentTests.Inrush"
  -t, --tags TEXT             Add one or multiple tags for the measurement adding one by one.
                              -Expected format: Enter one tag per -t option.
                              -Example to add 2 tags: "-t "test" -t "Internal"
  -v, --verbose               Enable verbose logging. Repeat to increase
                              verbosity.
  --help                      Show this message and exit.
```

Suggestion: 
- `help="The collection that this measurement belongs to. Collection names are specified using period-delimited namespace hierarchy and are case-insensitive. Example: 'CurrentTests.Inrush'"`
- `help="Tags describing the measurement. This option may be repeated to specify multiple tags. Example: '-t test -t internal'"`

Reply:
The \b\n allows  the -Expected and -Example on a different line. Facilitates readability. 

https://stackoverflow.com/questions/55389049/why-are-newlines-added-to-the-help-under-python-click/55389152#55389152

Reply:
Thanks, I didn't know that.